src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_x509chain.c

```diff
@@ -174,6 +174,8 @@ static void MergeStatusCodes(CFTypeRef key, CFTypeRef value, void* context)
         *pStatus |= PAL_X509ChainRevocationStatusUnknown;
     else if (CFEqual(keyString, CFSTR("MissingIntermediate")))
         *pStatus |= PAL_X509ChainPartialChain;
+    else if (CFEqual(keyString, CFSTR("UnparseableExtension")))
+        *pStatus |= PAL_X509ChainInvalidExtension;
```

Comment:
The InvalidAia test must be changed to show that this code is reported by both Windows and macOS (and, ideally, Linux),

Reply:
We tried to add assert that, but it failed on windows
`Assert.Contains(chain.ChainStatus, (status) => status.Status.HasFlag(X509ChainStatusFlags.InvalidExtension));`